daemon/container_windows.go

```diff
@@ -127,6 +127,7 @@ func populateCommand(c *Container, env []string) error {
 		FirstStart:     !c.HasBeenStartedBefore,
 		LayerFolder:    layerFolder,
 		LayerPaths:     layerPaths,
+		CPUShares:      c.hostConfig.CPUShares,
```

Comment:
@tiborvass That was me. I didn't notice the resources structure in there. Updated. Thanks for catching.